### PR TITLE
Task 364 from "Towards Controllable Biases in Language Generation"

### PR DESCRIPTION
**Pull request by Palipoor**
These are the related papers:
https://aclanthology.org/2020.findings-emnlp.291.pdf
https://arxiv.org/pdf/1909.01326.pdf

### CODE REVIEW

**Comment by danyaljj** [comment]
@XudongOliverShen wondering if you can help with this PR's review?

File: tasks/task364_regard_social_impact_classification.json [hunk 1]
```diff
@@ -0,0 +1,1842 @@
...
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, you will be given a sentence about a person. You should determine how the sentence affects how the person is perceived by most people in society. Your choices are: \nPositive: The social perception of [PERSON] in the sentence is considered predominantly positive.\nNegative: The social perception of [PERSON] in the sentence is considered predominantly negative.\nNo impact: There is no clear impact of social perception of [PERSON] associated with the sentence.",
+    "Positive Examples": [
+        {
+            "input": "[PERSON] behaved like a professional in the job interview.",
+            "output": "Positive",
+            "explanation": "Behaving professionaly in a job interview has a positive inpact on people's perception of a person."
```
Comment (XudongOliverShen) [inline tasks/task364_regard_social_impact_classification.json]: typo: inpact --> impact

**Review by XudongOliverShen — COMMENTED** [review]
Looks good to me!

**Review by swarooprm — COMMENTED** [review]
LGTM! Just make sure that labels across instances are near uniform and not skewed towards a particular label.

**Comment by danyaljj** [comment]
Thank you all! Merging!

**Pull request merged**